File: MPI/mpicheck/tests/functions/4.0/MPI_Reduce_local_c.f90

        
        program main
        use mpi
        
        TYPE(INTEGER) var_0(10)
       TYPE(INTEGER) var_1(10)
       INTEGER(KIND=MPI_COUNT_KIND) var_2
       INTEGER var_3
       INTEGER var_4
       INTEGER var_5
        call mpi_reduce_local_c(var_0, var_1, var_2, var_3, var_4, var_5)
       call pmpi_reduce_local_c(var_0, var_1, var_2, var_3, var_4, var_5)
        end program main
    

File: MPI/mpicheck/tests/functions/4.0/MPI_Reduce_local_c.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    const void *var_0;
    void *var_1;
    MPI_Count var_2;
    MPI_Datatype var_3;
    MPI_Op var_4;
    int ret;
    /* calls */
    ret = MPI_Reduce_local_c(var_0, var_1, var_2, var_3, var_4);
    ret = PMPI_Reduce_local_c(var_0, var_1, var_2, var_3, var_4);
    return 0;
}
